vault-github/run.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+BOLD='\033[0;1m'
+VAULT_ADDR=${VAULT_ADDR}
+VERSION=${VERSION}
+AUTH_PATH=${AUTH_PATH}
+GITHUB_EXTERNAL_AUTH_ID=${GITHUB_EXTERNAL_AUTH_ID}
+
+# Check if vault is installed
+if ! command -v vault &>/dev/null; then
+    printf "$${BOLD}Installing vault CLI ...\n\n"
+    # check if wget is installed
+    if ! command -v wget &>/dev/null; then
+        printf "wget is not installed. Please install wget in your image.\n"
+        exit 1
+    fi
+    # check if unzip is installed
+    if ! command -v unzip &>/dev/null; then
+        printf "unzip is not installed. Please install unzip in your image.\n"
+        exit 1
+    fi
```

Comment:
It would be nice to allow `curl` and `busybox unzip` as fallbacks if either wget/unzip aren't available. Perhaps a matter of future improvement. We could consider baking it into the agent as well (hidden command, `coder tools wget`, `coder tools unzip`, etc.)

Reply:
How can I fallback to the busybox unzip?